composer.json

```diff
@@ -36,9 +36,9 @@
         "mockery/mockery": "^1.0",
         "mongodb/laravel-mongodb": "^5.0",
         "orchestra/testbench": "^8.0|^9.0|^10.0",
-        "orchestra/testbench-core": "^8.0|^9.0|^10.0",
+        "orchestra/testbench-core": "^8.30|^9.7|^10.0",
         "phpstan/phpstan": "^1.0",
-        "phpunit/phpunit": "^10.0|^11.0",
+        "phpunit/phpunit": "^11.1",
```

Comment:
This is the first version the `--show-uncovered-for-coverage-text` option exists.